app/assets/stylesheets/layout/_header.scss

```diff
@@ -0,0 +1,46 @@
+.navbar {
+  background-color: $color-navbar-background;
+  box-shadow: 0 0 10px $color-text-normal;
+  height: 60px;
+
+  .navbar-toggler {
+    .navbar-toggler-icon {
+      background-image: url("data:image/svg+xml;charset=utf8,%3Csvg viewBox='0 0 32 32' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath stroke='rgb(255,255,255)' stroke-width='2' stroke-linecap='round' stroke-miterlimit='10' d='M4 8h24M4 16h24M4 24h24'/%3E%3C/svg%3E");
+    }
+  }
+}
+
+.navbar-brand {
+  color: $color-text-accent;
+  height: 100%;
+
+  &:hover {
+    color: $color-primary-background;
+  }
+}
+
+.navbar-logo {
+  height: 100%;
+  width: auto;
+}
+
+.navbar-nav {
+  @include media-breakpoint-up(md) {
+      display: flex;
```

Comment:
4 spaces?
It should be indented with 2 spaces

Reply:
Quite true! I was about to merge :derp: :v But you're comment stopped me. Fixing :loading: :entusiasmo: